apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -199,6 +166,41 @@ class ManageStudentsTable extends Component {
     return loginType === SectionLoginType.word || loginType === SectionLoginType.picture || loginType === SectionLoginType.email;
   };
 
+  // Cell formatters.
```

Comment:
For context: the `passwordFormatter` function was previously outside of the `PasswordReset` component. Now that we need access to `sectionId` on `props`, I moved it into the component.